### ISSUE TITLE
Paradigms of Interdependence

### ISSUE THREAD

**Issue by LIYINGZHEN** [LIYINGZHEN/book-reading#14]

**Comment by LIYINGZHEN** [comment]
- Private Victory precedes Public Victory. Self-mastery and self-discipline are the foundation of good relationships with others.

## SIX MAJOR DEPOSITS

1. Understanding the Individual

> Do unto others as you would have others do unto you.

2. Attending to the Little Things

The little kindnesses and courtesies are so important. Small discourtesies, little unkindnesses, little forms of disrespect make large withdrawals. In relationships, the little things are the big things.

3. Keeping Commitments

Keeping a commitment or a promise is a major deposit; breaking one is a major withdrawal.

4. Clarifying Expectations

The deposit is to make the expectations clear and explicit in the beginning.

5. Showing Personal Integrity

One of the most important ways to manifest integrity is to be loyal to those who are not present.

6. The Laws of Love and the Laws of Life

In other words, when we truly love others without condition, without strings, we help them feel secure and safe and validated and affirmed in their essential worth, identity, and integrity.
